src/scripts/CMSSync/index.ts

```diff
@@ -0,0 +1,103 @@
+import fs from 'fs'
+import moment from 'moment'
```

Comment:
Please avoid using moment. [It is no longer active](https://momentjs.com/docs/#/-project-status/). I recommend https://date-fns.org/

Reply:
Oh good to know, I used the moment because it was already installed, but I’m going to use this other one.